packages/debug/src/browser/style/debug.css

```diff
@@ -0,0 +1,19 @@
+/********************************************************************************
+ * Copyright (C) 2018 Ericsson and others.
```

Comment:
Could you please adjust the header to include the current year?
And feel free to add your name (e.g., [as in this file](https://github.com/eclipse-theia/theia/blob/master/packages/workspace/src/node/default-workspace-server.spec.ts)) or company whichever you prefer.